lab-anthony/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', ['test', 'lint']);
+
+gulp.task('test', function() {
+  return gulp.src(['test/read-test.js'])
+            .pipe(mocha());
+});
+
+gulp.task('lint', function() {
+  return gulp.src(['*.js', 'test/*.js', 'lib/*.js'])
+            .pipe(eslint())
+            .pipe(eslint.format());
+});
+
+gulp.task('watch', function(){
+  gulp.watch(['*.js', '/**/*.js', '/**/lib/*.js', '/**/test/*.js'], ['default']);
```

Comment:
The values at indices 2 and 3 in this watch array are included in the value at index 1.